ee-9/feature-pack/pom.xml

```diff
@@ -534,6 +535,10 @@
                                 <exclude>org.wildfly:wildfly-clustering-infinispan-client\z</exclude>
                                 <exclude>org.wildfly:wildfly-clustering-infinispan-extension\z</exclude>
                                 <exclude>org.wildfly:wildfly-clustering-web-infinispan\z</exclude>
+                                <!-- Don't transform Messaging modules -->
+                                <exclude>org.hornetq:.\z</exclude>
+                                <exclude>org.wildfly:wildfly-messaging-activemq-subsystem\z</exclude>
+                                <exclude>org.wildfly:wildfly-messaging-activemq-subsystem\z</exclude>
```

Comment:
These last two are the same, plus I don't think they'd be getting transformed anyway as you exclude them at L751 below.